### PR TITLE
Add fuzzer for arbitrary graph execution

### PR DESCRIPTION
**Pull request by DavidKorczynski**
CC @fcoUnda @learning-to-play

### CODE REVIEW

File: tensorflow/core/common_runtime/graph_constructor_fuzz.cc [hunk 1]
```diff
@@ -28,5 +29,40 @@ void FuzzImportGraphDef(const GraphDef& graph_def) {
 }
 FUZZ_TEST(GraphDefFuzz, FuzzImportGraphDef);
 
+void FuzzGraphEndToEnd1(const GraphDef& graph_def) {
```
Comment (fcoUnda) [inline tensorflow/core/common_runtime/graph_constructor_fuzz.cc:32]: nit: do we need that one at the end? If there is a duplicate, could you change the name to describe the difference?

File: tensorflow/core/common_runtime/BUILD [hunk 2]
```diff
@@ -3355,3 +3355,20 @@ tf_cc_fuzz_test(
         "//tensorflow/core/ops",
     ],
 )
+
```
Comment (fcoUnda) [inline tensorflow/core/common_runtime/BUILD:3358]: This target already exists right above, you just need to update deps, I'm surprised bazel doesn't complain.

**Review by fcoUnda — COMMENTED** [review]
Looks good, just a couple of comments!

File: tensorflow/core/common_runtime/BUILD (hunk 2 above)
Comment (DavidKorczynski) [reply tensorflow/core/common_runtime/BUILD:3358]: Sorry, this was a mistake on my end -- we only need the latter of them.

**Review by fcoUnda — APPROVED** [review]

**Review by fcoUnda — APPROVED** [review]

**Comment by DavidKorczynski** [comment]
Is there any more for me to do here?

**Pull request merged**